src/views/Space/Treasury.vue

```diff
@@ -31,6 +35,24 @@ const sortedAssets = computed(() =>
   })
 );
 
+const modalState: Ref<{
+  sendToken?: any;
+}> = ref({});
+const modalOpen = ref({
+  sendToken: false
+});
+
+function openModal(type: 'sendToken') {
+  modalState.value[type] = null;
+  modalOpen.value[type] = true;
+}
+const proposalsStore = useProposalsStore();
+function addTx(tx: TransactionType) {
```

Comment:
We should have consistent spacing between code block, ideally the init of the composables should appear on same place here: https://github.com/snapshot-labs/sx-ui/pull/392/files#diff-39b9c026880a261c912330b7d4de0b73817a8b8d3708e3199ecee4251e24d2e7R20
Then the functions appear after, separated with a new line